docs/docs/development.md

```diff
@@ -99,33 +99,13 @@ To install a specific version of Emscripten (e.g. `2.0.6`):
 
 ### `perspective-jupyterlab`
 
-To install the Jupyterlab plugin from your local working directory, give
-`jupyter labextension install` the path to the `perspective-jupyterlab` package:
-
-```bash
-jupyter labextension install ./packages/perspective-jupyterlab
-```
+To install the Jupyterlab/Jupyter Notebook plugins from your local working
+directory, simply install `python/perspective` with `pip` as you might normally
```

Comment:
I don't think this is correct, we're referring here to the local `python/perspective` folder relative to the root directory, not the separate `perspective-python` pypi-published package. lmk if this can be clearer